frontend/src/components/LoginPage.tsx

```diff
@@ -2,20 +2,22 @@ import React, { useState } from "react";
 import { RootSchema } from "../store/root-schema";
 import { connect } from "react-redux";
 import { Dispatch } from "../store";
-import { connectWebSocket, setUsername } from "../store/actions";
+import { connectWebSocket, setUsername, changeView } from "../store/actions";
 import { Jumbotron, Container, Form, Input, Button, FormGroup, Row } from "reactstrap";
 import { GameType, WebsocketOptions } from "../models/game-state.model";
 
 interface Props {
   username: string;
   connecting: boolean;
+  changeView: (username: string) => void;
```

Comment:
I'd rename this in component to `showStatistics`. Because that's what it does. The action in reducer is changing view, in here it is showingStatistics.

Reply:
Then we would need another action to go back to login. I think this one is more generic.

Reply:
You're right. I'm just talking about component here. You can easily name functiion `showStatistics` in here, and leave `changeView` in reducer, and everywhere else, that's what I mean.

Reply:
Ok, got it, you are right